### DISCUSSION TITLE
Account/Login Redirects Back to AccountLogin View

### DISCUSSION THREAD

**Discussion by muhammedyaman** [serenity-is/Serenity#6666] [Help]
Hi. Account/Login Redirects Back to AccountLogin View after using [this](https://serenity.is/docs/migration/mvc-to-core) guide to upgrade mvc application to core.  Can anyone help?

**Comment by muhammedyaman** [comment]
I managed to solve this by re-migrating using the same document. I don't know what caused this but it must be my fault since it worked in the second migration :)

**Comment by ali-h2010** [comment]
Thanks for the update. I was able to return a redirect instead of a view in the dashboard controller which solved the problem for me
